From c20a790b7bc0d5b9a6118ea4fd3e2c327fa2795d Mon Sep 17 00:00:00 2001
From: Robert Chiras <robert.chiras@nxp.com>
Date: Tue, 25 Sep 2018 15:49:08 +0300
Subject: [PATCH 4764/5242] MLK-19700-1: arm64: dts: imx8mq-evk: Limit LCDIF
 max resolution

commit  3141aecbfb177e068046da94d8016717dfba5058 from
https://source.codeaurora.org/external/imx/linux-imx.git

Limit the LCDIF maximum resolution to 1280x720 when used with ADV7535,
since the system banwidth does not allow higher resolutions to be used
with LCDIF in some stress scenarios.

Signed-off-by: Robert Chiras <robert.chiras@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../freescale/fsl-imx8mq-evk-lcdif-adv7535.dtsi    |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk-lcdif-adv7535.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk-lcdif-adv7535.dtsi
index afe53f1..c93885d 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk-lcdif-adv7535.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk-lcdif-adv7535.dtsi
@@ -36,7 +36,7 @@
 
 &lcdif {
 	status = "okay";
-	max-res = <1920>, <1080>;
+	max-res = <1280>, <720>;
 
 	port@0 {
 		lcdif_mipi_dsi: mipi-dsi-endpoint {
-- 
1.7.9.5

